.gitignore

```diff
@@ -15,3 +15,4 @@ docs/m.css
 *.so
 *__pycache__
 .pytest_cache
+dist
```

Comment:
On the long term, you can consider using the official gitignore for Python:

https://github.com/github/gitignore/blob/master/Python.gitignore